specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1164,6 +1236,430 @@
+        "createdBy": {
+          "type": "string",
+          "description": "Someone who created the incident."
+        },
+        "impactStartTime": {
+          "type": "string",
+          "description": "Impact start time for the incident.",
+          "format": "date-time"
+        },
+        "acknowledgeBy": {
+          "type": "integer",
+          "description": "Acknowledgement id for the incident.",
+          "format": "int64"
+        },
+        "incidentManagerAlias": {
+          "type": "string",
+          "description": "Incident manager alias for the incident."
+        },
+        "acknowledgeTime": {
+          "type": "string",
+          "description": "Acknowledge time for the incident.",
+          "format": "date-time"
+        },
+        "mitigateData": {
+          "type": "object",
+          "description": "Mitigate data for the incident.",
+          "properties": {
+            "mitigatedBy": {
+              "type": "string",
+              "description": "Contact who mitigated the incident."
+            },
+            "mitigateTime": {
+              "type": "string",
+              "description": "Time when the incident was mitigated.",
+              "format": "date-time"
+            }
+          }
+        },
+        "resolveData": {
+          "type": "object",
+          "description": "Resolve data for the incident.",
+          "properties": {
+            "resolvedBy": {
+              "type": "string",
+              "description": "Contact who resolved the incident."
+            },
+            "resolveTime": {
+              "type": "string",
+              "description": "Time when the incident was resolved.",
+              "format": "date-time"
+            }
+          }
+        }
+      }
+    },
+    "incidentBridge": {
+      "description": "Incident bridge information.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer",
+          "description": "Bridge id.",
+          "format": "int64"
+        },
+        "incidentId": {
+          "type": "integer",
+          "description": "Incident id.",
+          "format": "int64"
+        },
+        "bridgeURI": {
+          "type": "string",
+          "description": "Bridge uri."
+        },
+        "bridgeConfId": {
+          "type": "string",
+          "description": "Bridge conference id."
+        },
+        "expirationTime": {
+          "type": "string",
+          "description": "Expiration time of this bridge.",
+          "format": "date-time"
+        },
+        "phoneNumber": {
+          "type": "string",
+          "description": "Phone number of this bridge."
+        },
+        "bridgeType": {
+          "type": "string",
+          "description": "Bridge type."
+        },
+        "bridgeCreationMode": {
+          "type": "string",
+          "description": "Bridge creation mode.",
+          "enum": [
+            "Auto",
+            "Manual"
+          ]
```

Comment:
You'll need add "x-ms-enum" annotation. You might want to use an extensible enum modeling; read more here: [https://github.com/microsoft/api-guidelines/blob/e0ba179c0a066f96657a5d24e952c28f87ac0ca8/azure/Guidelines.md#use-extensible-enums](https://github.com/microsoft/api-guidelines/blob/e0ba179c0a066f96657a5d24e952c28f87ac0ca8/azure/Guidelines.md#use-extensible-enums)

Reply:
addressed